packages/web-components/update-scss-paths.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+declare -a component_names
+
+# Function to search and replace the string with proper escaping
+search_and_replace_string() {
+  directory="$1"
+  search_string="$2"
+  replace_string="$3"
+  
+  # search for scss files under directory
+  find "$directory" -type f -name "*.scss" ! -path "*/__stories__/*" | while read -r scss_file; do
+    # search and replace the string in the file
+    sed -i "" "s#$search_string#$replace_string#g" "$scss_file"
+		echo "Replaced imports in $scss_file"
+  done
+}
+
+# monitor for file changes under the `styles` directory
+fswatch -0 ../styles/ | while read -d "" changed_file
```

Comment:
does `fswatch` need to be installed first? when I run the script the first line of the output is

<img width="524" alt="image" src="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/8265238/d5d9b967-a0e1-4920-b4d6-141974b41521">

and then I believe the `cleanup` function runs

Reply:
Yes, `brew install fswatch` should make it work, forgot to add that in the PR description!